docs/Pflichtenheft.tex

```diff
@@ -30,7 +31,10 @@ \section{Einleitung}
 \Blindtext[1]
 
 \section{Zielbestimmung}
-\Blindtext[1]
+Der Systemadapter dient dem Ziel, die Funktion von OPC UA zu demonstrieren, ohne eine konkrete Produktionsanlage benutzen zu müssen. Somit kann beim Kunden das Protokoll flexibler vorgestellt werden. Hierzu soll das Programm einerseits eine Produktionsanlage simulieren und den Status grafisch darstellen, als auch eine Überwachungskonsole bereitstellen, die über OPC UA Werte der Anlage abfragt. \\
```

Comment:
Wie wärs wenn wir die Zeilen bei etwas 120 Zeichen umbrechen? Sonst sind Änderungen in den Dokumenten schwer nachvollziehbar.